tests/checkpoint/test_checkpoint_with_fluent_datasources.py

```diff
@@ -1088,12 +1088,43 @@ def test_newstyle_checkpoint_instantiates_and_produces_a_validation_result_when_
     assert result["success"]
 
 
+@pytest.mark.integration
+def test_newstyle_checkpoint_instantiates_and_produces_a_validation_result_when_run_validations_batch_request_sql_asset_in_checkpoint_run_sqlalchemy(
+    titanic_data_context_with_fluent_pandas_and_sqlite_datasources_with_checkpoints_v1_with_empty_store_stats_enabled,
+    common_action_list,
+    sa,
+):
+    context: FileDataContext = titanic_data_context_with_fluent_pandas_and_sqlite_datasources_with_checkpoints_v1_with_empty_store_stats_enabled
```

Comment:
Wanting to check, is there a reason why checkpoint_v1 is used here? Isn't that the one with Validation Operators?

Reply:
@Shinnnyshinshin This `v1` indicates actually the "non `LegacyCheckpoint`" because `LegacyCheckpoint` was experimental and did not have the notion of a "version" -- we created the notion of "version" in what we now call `Checkpoint` and assigned to it `version = 1` -- hence the fixtures are named to reflect that designation.  Thanks.

Reply:
Ok that's great. Is this understanding documented somewhere? It's new to me.

Reply:
@Shinnnyshinshin It has been our practice -- by default -- in the naming (without a formal document) since approximately 2020 when `Checkpoint` (the current design) was introduced (along the well-known migration scripts and documentation).  Thanks.